src/applications/ask-va/config/schema-helpers/personalInformationHelper.js

```diff
@@ -35,14 +35,25 @@ const ssnServiceInfo = (
   </>
 );
 
-const validateGroup = (errors, values) => {
-  if (!Object.keys(values).some(key => values[key])) {
-    errors.addError(`Please provide an answer`);
-  }
-};
+// const validateGroup = (errors, values) => {
+//   if (!Object.keys(values).some(key => values[key])) {
+//     errors.addError(`Please provide an answer`);
+//   }
+// };
 
-const validateSSandSNGroup = (errors, values) => {
-  if (!Object.keys(values).some(key => values[key])) {
+const validateSSandSNGroup = (errors, values, formData) => {
+  if (
+    !(
+      (formData.whoIsYourQuestionAbout === 'Someone else' &&
+        formData.relationshipToVeteran ===
+          "I'm connected to the Veteran through my work (for example, as a School Certifying Official or fiduciary)") ||
```

Comment:
Future note to start pulling from constants file for these instead of matching to strings. Less maintenance going forward.